semantic_conventions.md

```diff
@@ -22,6 +22,9 @@ Span tags apply to **the entire Span**; as such, they apply to the entire timera
 | `db.type` | string | Database type. For any SQL database, `"sql"`. For others, the lower-case database category, e.g. `"cassandra"`, `"hbase"`, or `"redis"`. |
 | `db.user` | string | Username for accessing database. E.g., `"readonly_user"` or `"reporting_user"` |
 | `error` | bool | `true` if and only if the application considers the operation represented by the Span to have failed |
+| `graphql.operation_type` | string | The type of graphql query operation for the associated Span. E.g., `"query"`, `"mutation"`, `"subscription"` |
+| `graphql.operation_name` | string | The name of graphql query operation for the associated Span. E.g., given `query FindFsers { users { name } }` then the operation name is `"FindUsers"` |
```

Comment:
Wouldn't this usually match the span's operation name, if there's one span per query?

Reply:
potentially - yes

Reply:
Then I'd rather leave it to the instrumentation library to deal with, instead of specifying it here. This way, a custom tag can be set if the GraphQL operation name is *not* the same as the span's.